package.json

```diff
@@ -50,68 +45,63 @@
     "precss": "=1.4.0",
     "raw-loader": "=0.5.1",
     "react": "=15.5.4",
+    "react-docgen": "=2.19.0",
+    "react-docgen-displayname-handler": "=1.0.1",
+    "react-docgen-typescript": "=1.0.2",
     "react-dom": "=15.5.4",
+    "react-styleguidist": "=6.0.32",
+    "react-styleguidist-visual": "=0.1.4",
     "react-test-renderer": "=15.5.4",
     "rimraf": "=2.6.1",
     "serve": "=5.1.4",
-    "storybook-readme": "=3.0.6",
     "style-loader": "=0.16.1",
     "svgcomp": "=1.0.1",
     "svgo": "=0.7.2",
     "ts-jest": "=20.0.6",
-    "ts-loader": "=2.0.3",
+    "ts-loader": "=3.1.0",
     "tslint": "=5.6.0",
     "tslint-config-airbnb": "=5.2.1",
     "tslint-react": "=3.2.0",
-    "typescript": "=2.5.2"
+    "typescript": "=2.5.2",
+    "webpack": "=3.8.1"
   },
   "peerDependencies": {
     "react": "^15.0.0",
     "react-dom": "^15.0.0",
     "typescript": "^2.5.0"
   },
   "scripts": {
-    "clean:storybook": "rimraf build/storybook/*",
     "clean:build": "rimraf build/*",
     "clean:dist": "rimraf dist/*",
+    "clean:docs": "rimraf build/docs/*",
+    "clean:icons": "rimraf build/icons/* src/components/Icon/icons/*.tsx",
     "clean": "run-p clean:build clean:dist",
-    "copy:story-assets": "cpx \"src/stories/assets/**/*\" \"dist/stories/assets\"",
-    "copy:readmes": "cpx \"src/components/**/README.md\" \"dist/components\"",
     "build:css:global": "postcss src/css/index.css --output dist/yamui-base.css",
     "build:css:components": "postcss \"src/components/**/*.css\" --dir \"dist/components\" --base \"src/components\"",
     "build:ts:components": "tsc --project .",
-    "build": "run-s clean:dist clean:build copy:** build:**",
+    "build": "run-s clean build:**",
+    "watch:css:global": "postcss src/css/index.css --output dist/yamui-base.css --watch",
+    "watch:css:components": "postcss \"src/components/**/*.css\" --dir \"dist/components\" --base \"src/components\" --watch",
+    "watch:ts:components": "tsc --project . --watch",
+    "watch": "run-p watch:**",
     "test:eslint:root": "eslint \"./*.+(js|jsx)\"",
     "test:eslint:sources": "eslint \"+(src|config)/**/*.+(js|jsx)\"",
     "test:eslint": "run-p test:eslint:*",
     "test:tslint": "tslint --type-check --project tsconfig.json",
-    "test:mdlint": "markdownlint README.md assets/README.md assets/Icons/README.md src/components/README.md",
+    "test:mdlint": "markdownlint README.md assets/README.md assets/Icons/README.md",
     "test:unit": "jest --config config/jest/config.json",
-    "test:visual:build": "run-s clean:storybook build storybook:static:build",
-    "test:visual:serve": "run-p storybook:static:serve test:visual:test-and-stop",
-    "test:visual:test-and-stop": "run-s --continue-on-error visual:test storybook:static:stop",
-    "test:visual": "run-s test:visual:build test:visual:serve",
+    "test:visual:build": "styleguidist build --config config/styleguide/config.visualdiff.js",
```

Comment:
@unindented i think you and i had some merge conflicts here. since styleguidist builds from webpack we shouldn't need any watch tasks now, and shouldn't need to run `build` as part of the styleguidist static build

Reply:
It's only needed when running `test:visual`, to generate the folder that then gets loaded by Puppeteer. So it looks like this now:

```
"test:visual": "run-s clean:docs test:visual:build test:visual:run"
```

Reply:
Aah you're saying I should remove all `watch` tasks! You're right!

Reply:
Ok, I think I understand now. Regarding "shouldn't need to run build as part of the styleguidist static build" I think you misread the command. It's calling `styleguidist` with the `build` command, not our `build` script.

Reply:
yes i did!